docs/overview/overview.md

```diff
@@ -1,45 +1,31 @@
 
-> [!NOTE]
-> If you're looking to develop an Office Add-In for Excel, please visit our [quick start](/office/dev/add-ins/quickstarts/excel-quickstart-jquery) or [tutorial](/office/dev/add-ins/tutorials/excel-tutorial). Learn more about the differences between Office Scripts and Office Add-ins in [the Differences between Office Scripts and Office Add-ins article](../resources/differences-scripts-add-ins.md).
+<img alt="A gif showing a script being recorded and replayed on different worksheets." src="../images/overview.gif" />
 
-## Script environment
-
-Office Scripts are written in either [TypeScript or JavaScript](#scripting-language-typescript-or-javascript) and use the [Office JavaScript APIs](#office-javascript-api) to interact with an Excel workbook.
-
-### Scripting language: TypeScript or JavaScript
-
-Office Scripts are written in TypeScript or JavaScript. The Action Recorder generates code in TypeScript (which is a superset of JavaScript). Office Scripts documentation uses TypeScript, but if you're more comfortable with JavaScript, you can use that instead.
-
-Office Scripts are largely self-contained pieces of code. Only a small part of TypeScript's functionality is used. Therefore, you can edit scripts without having to learn the intricacies of the TypeScript. The Code Editor also handles the installation, compilation, and execution of code, so you don't need to worry about anything but the script itself. It's possible to learn the language and create scripts without previous programming knowledge. However, if you're new to programming, we recommend learning some fundamentals before proceeding with Office Scripts:
-
-- Learn the basics of JavaScript. You should feel comfortable with concepts like variables, control flow, functions, and data types. [Mozilla offers a good, comprehensive tutorial on JavaScript](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Introduction).
-- Learn about types in TypeScript. TypeScript builds on JavaScript by ensuring at compile-time the right types are used for method calls and assignments. The TypeScript documentation on [interfaces](https://www.typescriptlang.org/docs/handbook/interfaces.html), [classes](https://www.typescriptlang.org/docs/handbook/classes.html), [type inference](https://www.typescriptlang.org/docs/handbook/type-inference.html), and [type compatibility](https://www.typescriptlang.org/docs/handbook/type-compatibility.html) will be the most useful.
-
-### Office JavaScript API
+## Action Recorder
 
-Office Scripts uses the Office JavaScript API. Specifically, they use the APIs available in the [ExcelApi Online requirement set](/javascript/api/excel). Your scripts have access to most of the features in the Excel namespace. The exceptions are listed in the [Differences between Office Scripts and Office Add-ins](../resources/differences-scripts-add-ins.md#apis) article.
+![The Action Recorder after recording several actions.](../images/action-recorder-intro.png)
```

Comment:
Another thing. The images on [this page](https://review.docs.microsoft.com/en-us/office/dev/scripts/overview/overview?branch=AlexJ-Overview) don't seem to meet the [screenshot standards](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-create-screenshot?branch=master).